cli/dcoscli/data/help/package.txt

```diff
@@ -5,24 +5,22 @@ Usage:
     dcos package --config-schema
     dcos package --help
     dcos package --info
-    dcos package describe [--app --cli --config]
-                          [--render]
-                          [--package-versions]
-                          [--options=<file>]
+    dcos package describe <package-name>
+                          [--app --cli --config --render --options=<file>]
```

Comment:
I looked at the implementation and made some change to this line to reflect the implementation. It is still pretty complicated. I'll look into simplifying this when we re-implement this in 1.10 under the experimental features.